composer.json

```diff
@@ -37,11 +38,28 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "DotTest\\AnnotatedServices\\": "tests"
+            "DotTest\\AnnotatedServices\\": "test"
         }
     },
+    "scripts": {
+        "check": [
+            "@cs-check",
+            "@test"
+        ],
+        "cs-check": "phpcs",
+        "cs-fix": "phpcbf",
+        "test": "phpunit --colors=always",
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml",
+        "static-analysis": "psalm --shepherd --stats"
+    },
     "suggest": {
```

Comment:
Please remove the `suggest` section completely, becuase the suggested packages are already in the `require` section.